### PR TITLE
Update data.txt

### PR DESCRIPTION
**Pull request by hellogoodbye0101**
Thx

### CODE REVIEW

**Comment by triketora** [comment]
Hi @hellogoodbye0101, can you include information in the commit message about you and the source of your information? There's some guidance in the contributor guidelines. Thanks!

**Comment by hellogoodbye0101** [comment]
LinkedIn.

**Comment by triketora** [comment]
Can you update the commit message with that information?

File: data.txt [hunk 1]
```diff
@@ -118,8 +118,8 @@
   last_updated: 2016-12-18
 [reddit]
   company: Reddit
-  num_female_eng: 2
-  num_eng: 14
+  num_female_eng: 3
```
Comment (hellogoodbye0101) [inline data.txt]: Linkedin is source. I also work there.

File: data.txt [hunk 2]
```diff
@@ -118,8 +118,8 @@
...
+  num_eng: 66
```
Comment (hellogoodbye0101) [inline data.txt]: Sourced via LinkedIn.

Comment (triketora) [inline data.txt]: Can you put this in the commit message please?

Comment (triketora) [inline data.txt]: On further thought -- LinkedIn isn't a very reliable source of data. Do you have a better source or confirmation otherwise?

Comment (hellogoodbye0101) [inline data.txt]: Yes.

Comment (hellogoodbye0101) [inline data.txt]: I can't abide by your contribution terms.

**Pull request closed without merging**